packages/components/inputs/rich-text-utils/src/plugins/mark.tsx

```diff
@@ -1,6 +1,25 @@
 import { isKeyHotkey } from 'is-hotkey';
 import { warning } from '@commercetools-uikit/utils';
 import memoize from 'lodash/memoize';
+import type { ReactNode } from 'react';
+import type { TEditor, TMark } from '../editor.types';
+
+type TMarkPluginOptions = {
+  hotkey: string;
+  typeName: string;
+  RenderMark: ReactNode;
+  command: string;
+  query: string;
+  [option: string]: string | ReactNode;
+};
+
+type TProps = {
```

Comment:
```suggestion
type TRenderMarkProps = {
```
Maybe we could use a more specific name.